src/test/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImplTest.java

```diff
@@ -50,6 +48,46 @@
     private static final int FIRST_PAYEE_LINE_IN_OUTPUT_FILE = HEADER_LINE_IN_OUTPUT_FILE + NOTIFICATION_EMAIL_ADDRESSES.length + 1;
     private static final Logger LOG = LogManager.getLogger(CuExtractPaymentServiceImplTest.class);
     
+    private static final String EDOC_DV_IDENTIFIER_AND_NUMBER = CuDisbursementVoucherConstants.DV_EXTRACT_EDOC_NUMBER_PREFIX_IDENTIFIER + "12345678 ";
+    private static final int MAX_NUM_DV_CHECK_STUB_NOTES_TESTS = 4;
+    private static final int MAX_NUM_DV_CHECK_STUB_LINES_PER_TEST = 3;
+    
+                                                       //          1         2         3         4         5         6         7
+                                                       //01234567890123456789012345678901234567890123456789012345678901234567890
+    private static final String[] DV_NOTES_TEST_DATA = {"First line of note text present. Next two lines blank.",
+                                                        "",
+                                                        "",
+                                                        
+                                                        "First line for note text fully filled with data to see what will happen", 
+                                                        " Second line partially filled. Third is blank",
+                                                        "",
+                                                        
+                                                        "First line notes text fully filled with characters to see how this wrap", 
+                                                        "Second line notes text is fully filled with data to see what it will do",
+                                                        "Third line is parially filled.",
```

Comment:
Minor typo in line text; should say "...partially..." instead. The same goes for the matching expected result.

Reply:
fixed